### ISSUE TITLE
problem with plotdom, object 'chr' not found

### ISSUE THREAD

**Issue by Meowth-3Dchr** [wbaopaul/GMAP#15]
Hello,

I got the following error when I use plotdom function, could you help me with this? Thank you!
> pp = plotdom(FLC_normalized_matrix, index_obj = index_obj_mm9_10k_hicpro, res$hierTads, 16, start_bin=212671, end_bin=222502, cthr=30, resl=10000)
Error in `[.data.frame`(index_obj, chr == chr0) : object 'chr' not found

**Comment by wbaopaul** [comment]
I guess you need to specify the chromosone name and the relative start and end bin of the selected chromosome. For example, if you want to plot region of chr1: 50,000,000-52,000,000, you need to use parameter as follow:
chr0=chr1, start_bin=5000, end_bin=5200

Hope it helps,
Wenbao

**Comment by Meowth-3Dchr** [comment]
Hello Wenbao,

Thanks for your reply! I tried "pp = plotdom(FLC_normalized_matrix, index_obj = index_obj_mm9_10k_hicpro, res$hierTads, chr0=chr16, start_bin=0, end_bin=9831, cthr=30, resl=10000)", but still got error "Error in plotdom(FLC_normalized_matrix, index_obj = index_obj_mm9_10k_hicpro,  : 
  object 'chr16' not found".
I also tried the following still have same error:
chr0='chr1'
chr0="chr1"
"chr1"
'chr1'

Another question is the start_bin is the coordinate where the bin start and end_bin is the coordinate the bin end, but not the index of the bin, right? For example, if I want to plot the whole chromosome 16 of mm9, so I should put 0 as the value for start_bin and I should put 9831 as the end_bin value, since the length of chr16 is 98319150, , but not the index 212671 for start bin and ,222502 for end_bin?
Following shows the binning info of chr16 in hicpro's *_abs.bed file
> head(chr16_index)
     V1    V2    V3     V4
1 chr16     0 10000 212671
2 chr16 10000 20000 212672
3 chr16 20000 30000 212673
4 chr16 30000 40000 212674
5 chr16 40000 50000 212675
6 chr16 50000 60000 212676
> tail(chr16_index)
        V1       V2       V3     V4
9827 chr16 98260000 98270000 222497
9828 chr16 98270000 98280000 222498
9829 chr16 98280000 98290000 222499
9830 chr16 98290000 98300000 222500
9831 chr16 98300000 98310000 222501
9832 chr16 98310000 98319150 222502

Thank you very much!
